### PR TITLE
Update INSTALL.md

### PR DESCRIPTION
**Pull request by Everlastinggit**
Alacritty avaible in official repo so I removed the mention of the copr repo.

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -4,13 +4,6 @@ For distributions that do not ship Alacritty officially, there are some third
 party packages available. Only the cargo package is maintained officially, so
 proceed with caution.
 
-## [Fedora](https://copr.fedorainfracloud.org/coprs/pschyska/alacritty)
```
Comment (kchibisov) [inline INSTALL.md:7]: should remove third party repo mention, since there should be none,

**Review by kchibisov — CHANGES_REQUESTED** [review]

File: INSTALL.md [hunk 2]
```diff
@@ -1,10 +1,4 @@
-# Third Party Repositories
```
Comment (Everlastinggit) [inline INSTALL.md:1]: Removed mention of third party repositories.

File: INSTALL.md [hunk 3]
```diff
@@ -1,10 +1,4 @@
...
-
-For distributions that do not ship Alacritty officially, there are some third
-party packages available. Only the cargo package is maintained officially, so
-proceed with caution.
-
-## Cargo
+# Cargo Insallation
```
Comment (Everlastinggit) [inline INSTALL.md:1]: Removed mention of Third Party Repositories and changed Cargo section in Cargo Installation section.

**Review by kchibisov — APPROVED** [review]

File: INSTALL.md (hunk 3 above)
Comment (xentec) [reply INSTALL.md:1]: Minor typo: Cargo Ins_t_allation.

**Pull request merged**